src/sentry/snuba/metrics/utils.py

```diff
@@ -328,15 +330,81 @@ class OrderByNotSupportedOverCompositeEntityException(NotSupportedOverCompositeE
     ...
 
 
-def get_intervals(start: datetime, end: datetime, granularity: int, interval: Optional[int] = None):
+def to_intervals(
+    start: Optional[datetime], end: Optional[datetime], interval_seconds: int
+) -> Tuple[Optional[datetime], Optional[datetime], int]:
+    """
+    Given a start date, end date and an alignment interval in seconds returns the aligned start, end and
```

Comment:
```suggestion
    Given a `start` date, `end` date and an alignment interval in seconds returns the aligned start, end and
```
nit: just to highlight it's variable names in func params.